src/robotide/ui/mainframe.py

```diff
@@ -170,6 +171,7 @@ def __init__(self, application, controller):
         self._plugin_manager = PluginManager(self.notebook)
         self._review_dialog = None
         self._view_all_tags_dialog = None
+        self._curdir = None
```

Comment:
self._curdir means to save current external file dir right?

so a more specified name will be better, it probably will make us confused with current workspace dir